examples/mnist/train.py

```diff
@@ -68,12 +69,8 @@ def onehot(labels, num_classes=10):
   return x.astype(jnp.float32)
 
 
-def cross_entropy_loss(logits, labels):
-  return -jnp.mean(jnp.sum(onehot(labels) * logits, axis=-1))
-
-
 def compute_metrics(logits, labels):
-  loss = cross_entropy_loss(logits, labels)
+  loss = jnp.mean(optax.softmax_cross_entropy(logits, onehot(labels)))
```

Comment:
(Same comments as above: should we maybe remove `nn.log_softmax(x)`? And would it make sense to specify arguments by keyword for extra safety?)

Reply:
Removed nn.log_softmax(x)